dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -578,6 +614,44 @@ def test_categorical_merge(self):
         }
         self.assertCountEqual(report_count, expected_dict)
 
+        # Setting up of profile with stop condition not yet met
```

Comment:
in the above, should we check that stop condition is not met for cases that don't meet and the stopped at stay none?